ansible/roles/avocado/tasks/main.yaml

```diff
@@ -81,3 +81,13 @@
     - "{{avocado_repos}}"
   tags:
     - setup
+
+- name: Install BVT script
+  copy:
+    src: "{{playbook_dir}}/../scripts/host-os-bvt.py"
+    dest: /usr/bin/host-os-bvt
```

Comment:
This command is executed with sudo (because it is required by avocado with libvirt) and the root user does not have `/usr/local/bin` in the path by default, so I wanted to avoid the hassle of dealing with this. Do you think that's the proper way to do it?

Reply:
My root's PATH is defined with `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin` and I don't recall changing anything specific to PATH. Let's not worry about that. It doesn't change any function behaviour.